src/features/semanticModels/types.ts

```diff
@@ -30,7 +30,9 @@ export interface SemanticModelsInitialState {
   uploadError: string
   openApiLink: string
   openApiError: string
-  error: string
+  // Add an ESLint exception until there is a solution
+  // eslint-disable-next-line
+  error: any
```

Comment:
please define an interface for ErrorResponse containing all the attributes you like to access. Like here we use `error.response.status` 

```
export interface ErrorResponse {
  response: {
    status: number
  }
}

...

  error: ErrorResponse
```